weave/trace_server/migrations/023_add_annotation_queues.up.sql

```diff
@@ -0,0 +1,241 @@
+
+    /*
+    `queue_id`: String (UUID format), references annotation_queues.id.
+    */
+    queue_id String,
+
+    /*
+    `call_id`: References calls_merged.id. The actual call being annotated.
+    */
+    call_id String,
+
+    /*
+    `call_started_at`: Cached from call for sorting/filtering without joins.
+    Snapshot at add time.
+    */
+    call_started_at DateTime64(3),
+
+    /*
+    `call_ended_at`: Cached from call for sorting/filtering without joins.
+    Snapshot at add time.
+    */
+    call_ended_at Nullable(DateTime64(3)),
+
+    /*
+    `call_op_name`: Cached from call for sorting/filtering without joins.
+    Snapshot at add time.
+    */
+    call_op_name String,
+
+    /*
+    `call_trace_id`: Cached from call for sorting/filtering without joins.
+    Snapshot at add time.
+    */
+    call_trace_id String,
+
+    /*
+    `display_fields`: JSON paths to show annotator, e.g., ['input.prompt', 'output.text'].
+    Focuses annotator attention on relevant fields. Specified per batch when adding calls.
+    */
+    display_fields Array(String),
+
+    /*
+    `added_at`: Timestamp when this call was added to the queue.
+    */
+    added_at DateTime64(3) DEFAULT now64(3),
+
+    /*
+    `added_by`: W&B user ID who added this call to the queue.
+    */
+    added_by Nullable(String),
+
+    /*
+    `created_at`: Timestamp when the row was created.
+    */
+    created_at DateTime64(3) DEFAULT now64(3),
+
+    /*
+    `created_by`: W&B user ID who created this record.
+    */
+    created_by String,
+
+    /*
+    `updated_at`: Timestamp when the row was last modified.
+    */
+    updated_at DateTime64(3) DEFAULT now64(3),
+
+    /*
+    `deleted_at`: Soft delete timestamp. NULL means not deleted.
+    */
+    deleted_at Nullable(DateTime64(3))
+) ENGINE = MergeTree()
+ORDER BY (project_id, queue_id, id)
+SETTINGS
+    enable_block_number_column = 1,
+    enable_block_offset_column = 1;
+
+-- ============================================================================
+-- annotator_queue_items_progress: Per-annotator workflow state tracking
+-- ============================================================================
+CREATE TABLE annotator_queue_items_progress (
+    /*
+    `id`: String (UUID format), unique identifier for this progress record. This is the primary reference.
+    */
+    id String,
+
+    /*
+    `project_id`: The project identifier.
+    */
+    project_id String,
+
+    /*
+    `queue_item_id`: String (UUID format), references annotation_queue_items.id.
+    */
+    queue_item_id String,
+
+    /*
+    `queue_id`: String (UUID format), denormalized from queue_item for efficient querying without joins.
+    */
+    queue_id String,
+
+    /*
+    `call_id`: Denormalized from queue_item for efficient querying without joins.
+    */
+    call_id String,
+
+    /*
+    `annotator_id`: W&B user ID of the annotator. NULL when state is 'pending'.
+    */
+    annotator_id Nullable(String),
```

Comment:
recommend that this is non-nullable, which implies we do not insert a "empty" version of this for each row. This also implies that the `annotation_state` should have `in_progress`, not `pending` and mark as `in_progress` when it starts to avoid collision